skycatalogs/catalog_creator.py

```diff
@@ -884,7 +900,12 @@ def create_pointsource_catalog(self):
         -------
         None
         """
-        arrow_schema = make_star_schema()
+        inputs = {'star_truth': self._star_truth}
+        file_metadata = assemble_file_metadata(self._pkg_root,
+                                               inputs=inputs,
+                                               run_options=self._run_options)
+
+        arrow_schema = make_star_schema(metadata_input=file_metadata)
```

Comment:
Is the comment below,
>         #  Need a way to indicate which object types to include; deal with that
>         #  later.  For now, default is stars + sn
still relevant?  I would think not, since `sncosmo` support has been removed.  As it stands, it's hard to see how it relates to the code around it.

Reply:
Nope, no longer relevant.  Will delete.